learning_wavelets/data/datasets.py

```diff
@@ -11,26 +11,33 @@ def normalise(image):
     return (image / 255) - 0.5
 
 # data augmentation
-def random_rotate_image(image):
+def random_rotate_flip_image(image):
     # in numpy
-    image = ndimage.rotate(image, np.random.uniform(-30, 30), reshape=False)
+
+    rot_coef = np.random.randint(4, size=1)[0]  # 0-3
+    flip_coef = np.random.randint(2, size=1)[0]  # 0-1
+
+    image = np.rot90(image, rot_coef).copy()
+
+    if flip_coef == 1:
+        image = np.fliplr(image).copy()
+
     return image
 
-def tf_random_rotate_image(image):
+def tf_random_rotate_flip_image(image):
     # in tf
     im_shape = image.shape
-    [image,] = tf.py_function(random_rotate_image, [image], [tf.float32])
+    [image,] = tf.py_function(random_rotate_flip_image, [image], [tf.float32])
     image.set_shape(im_shape)
     return image
 
 # patch selection
-def select_patch_in_image_function(patch_size, seed=0):
+def select_patch_in_image_function(patch_size):
     def select_patch_in_image(image):
         if patch_size is not None:
             patch = tf.image.random_crop(
                 image,
-                [patch_size, patch_size, 1],
-                seed=seed,
+                [patch_size, patch_size, 1]
```

Comment:
```suggestion
                [patch_size, patch_size, 1],
```

Don't forget the `,` at the end of lines, this doesn't change anything to the Python run, but it allows an easier review of the changes.